imap_processing/cli.py

```diff
@@ -415,6 +418,12 @@ def pre_processing(self) -> ProcessingInputCollection:
         dependencies = ProcessingInputCollection()
         dependencies.deserialize(self.dependency_str)
         dependencies.download_all_files()
+
+        # Furnish spice kernels
+        kernel_paths = dependencies.get_file_paths(source=SPICESource.SPICE.value)
+        logger.info(f"Furnishing kernels: {kernel_paths}")
+        spiceypy.furnsh([str(kernel_path) for kernel_path in kernel_paths])
```

Comment:
Does spiceypy not accept Path objects?
Do you need to resolve the path before turning it to a string?